src/main/scala/ReleasePlugin.scala

```diff
@@ -103,17 +108,35 @@ object ReleasePlugin extends AutoPlugin {
       private val SkipTests = "skip-tests"
       private val CrossBuild = "cross"
       private val FailureCommand = "--failure--"
-      private val releaseParser = (Space ~> WithDefaults | Space ~> SkipTests | Space ~> CrossBuild).*
 
-      val releaseCommand: Command = Command(releaseCommandKey)(_ => releaseParser) { (st, args) =>
+      private val ReleaseVersion = "release-version"
+      private val releaseVersionStringParser: Parser[String] = ReleaseVersion
+      private val NextVersion = "next-version"
+      private val nextVersionStringParser: Parser[String] = NextVersion
+
+      private val releaseVersionParser = Space ~> releaseVersionStringParser ~ Space ~ StringBasic
+      private val nextVersionParser = Space ~> nextVersionStringParser ~ Space ~ StringBasic
+      private val releaseParser = (releaseVersionParser | nextVersionParser | Space ~> WithDefaults | Space ~> SkipTests | Space ~> CrossBuild).*
+
+      val releaseCommand: Command = Command(releaseCommandKey)(_ => releaseParser) { (st, args: Seq[Serializable]) =>
         val extracted = Project.extract(st)
         val releaseParts = extracted.get(releaseProcess)
         val crossEnabled = extracted.get(releaseCrossBuild) || args.contains(CrossBuild)
+
+        //args is a Seq[Serializable], the elements are  either ((String, List[String]),String) or String
```

Comment:
`Seq[Serializable]` is terrible

https://github.com/xuwei-k/sbt-release/commit/160f856b26e643ee57